src/js/mixins/evented.js

```diff
@@ -68,9 +88,9 @@ const isValidEventType = (type) =>
  * @param  {Object} target
  *         The object to test.
  */
-const validateTarget = (target) => {
-  if (!target.nodeName && !isEvented(target)) {
-    throw new Error('Invalid target; must be a DOM node or evented object.');
+const validateTarget = (target, obj, fnName) => {
```

Comment:
report the object name and the evented function it was trying to call when the error occurred. Should make understanding issues with this code easier to follow.